app/models/poll.rb

```diff
@@ -168,6 +168,54 @@ def respondent_question
     self.questions.where(collects_respondent: true).first
   end
 
+  def duplicate
+    duplicate = self.dup
```

Comment:
That's related to the poll's `recurrence`, which should fire the poll's questions regularly every X days (depending on the recurrence configured). As far as I understand, if the poll hasn't started yet, the `current_occurrence` should be blank. But be sure to double-check.